app/bundles/CampaignBundle/Model/CampaignModel.php

```diff
@@ -787,4 +794,43 @@ public function getCampaignIdsWithDependenciesOnEmail(int $emailId): array
     {
         return $this->getRepository()->getCampaignIdsWithDependenciesOnEmail($emailId);
     }
+
+    /**
+     * @param Campaign $entity
+     *
+     * @return array<string, array<int, array<string, int|string>>>
+     *
+     * @throws Exception
+     */
+    public function getCountryStats($entity, \DateTimeImmutable $dateFrom, \DateTimeImmutable $dateTo, bool $includeVariants = false): array
```

Comment:
We should be able to use the real types due to https://www.php.net/manual/en/language.oop5.variance.php
```suggestion
    /**
     * @return array<string, array<int, array<string, int|string>>>
     *
     * @throws Exception
     */
    public function getCountryStats(Campaign $entity, \DateTimeImmutable $dateFrom, \DateTimeImmutable $dateTo, bool $includeVariants = false): array
```